report/renderer/report-ui-features.js

```diff
@@ -38,6 +38,10 @@ function getTableRows(tableEl) {
   return Array.from(tableEl.tBodies[0].rows);
 }
 export class ReportUIFeatures {
+  static Events = {
+    refreshLighthouseReport: 'refreshLighthouseReport',
```

Comment:
This event name feels descriptive of our single use case listener but not the event itself right now.

`newLhrAvailable`
`lhrUpdated`
`localeUpdated`
`rerenderReportRequest`

do any of those strike a chord?

Reply:
removed the event, replaced with a callback